Framework/PythonInterface/plugins/algorithms/LoadSINQFile.py

```diff
@@ -68,8 +68,7 @@ def PyExec(self):
 
         if inst == "POLDI":
             if ws.getNumberHistograms() == 800:
-                ws.maskDetectors(SpectraList=list(range(0,800))[::2])
-
+                ws.maskDetectors(SpectraList=list(range(2, 800, 2)))
```

Comment:
In this case no, as it attempted to mask spectra `0, 2, 4...` `MaskDetectors` silently skipped spectra 0, it was only because we emitted a key error I noticed these problems.